client/src/global/components/list/Filters/Group/index.js

```diff
@@ -0,0 +1,77 @@
+import React, { useRef } from "react";
+import PropTypes from "prop-types";
+import Search from "../Search";
+import Filter from "../Filter";
+import * as Styled from "./styles";
+
+import withScreenReaderStatus from "hoc/withScreenReaderStatus";
+
+function FiltersGroup(props) {
+  const {
+    filters,
+    hideSearch = false,
+    updateFilterState,
+    onReset,
+    showReset,
+    setScreenReaderStatus,
+    className
+  } = props;
+
+  const searchInput = useRef(null);
+
+  const handleReset = () => {
+    onReset();
+    setScreenReaderStatus("Search and filters reset.");
+    if (searchInput.current) {
+      searchInput.current.focus();
+      searchInput.current.value = "";
+    }
+  };
+
+  const onSubmit = e =>
+    updateFilterState(e, "keyword", searchInput.current.value);
+
+  /* eslint-disable no-nested-ternary */
```

Comment:
Point taken re: disabling rules. Personally, I'm not a fan of the `no-nested-ternary` eslint rule. I think it isn't nuanced enough. For example, it doesn't distinguish actually nested ternaries, which I agree are hard to read, from merely chained ones, which I do think have a place in certain cases. In the `featuredLabel` case below, for instance, where we're choosing between three predefined strings, I think alternative forms— adding a function, using a `let` and a sequence of `ifs`— are worse patterns to implement. But, as the many github discussions of this rule would indicate, implementing a more nuanced version of this rule isn't easy.

Reply:
Fair point re: the nuance around this. I don't ever see you using chained ternaries beyond a single additional condition. The alternative I was thinking of was a `switch` statement, which serves just this purpose of returning a value when a condition isn't a simple if/else. It's also scales well. Like with `featuredLabel`, should you need to consider other models, you could add them as cases to the switch rather than additional ternary chains. Granted, a `switch` wouldn't really work for this `resetLabel` example, since you have multiple conditions. So it's not a one-size-fits-all solution.

Reply:
Totally. I used to write crazy ones, but I have tried to limit myself to using ternaries in max three option cases. And I agree that where they work, switches are an easier to read alternative.